webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -65,3 +68,21 @@ def inner(request, *args, **kwargs):
             raise
         return view(request, *args, **kwargs)
     return inner
+
+
+def track_api_pageview_decorator(view):
```

Comment:
It is. I guess I could do `from crashstats.base.ga import track_api_pageview as _track_api_pageview` or something and call the decorator `track_api_pageview` instead. What do you think?